utils/fake_libc_include/_fake_defines.h

```diff
@@ -227,3 +225,15 @@
 #define __m512d_u int
 #define __m512i   int
 #define __m512i_u int
+
+/* C11 stdnoreturn.h defines */
+#define __noreturn_is_defined 1
+#define noreturn _Noreturn
+
+/* C11 threads.h defines */
+#define thread_local _Thread_local
```

Comment:
Please add a specific parser test for `_Thread_local` too. The C-to-C tests are insufficient for this - they won't catch errors in building the AST in the parser.

Reply:
... and same comment for `static_assert`. Please test it both inside and outside a function, in `test_c_parser`.